PhysicsTools/PatAlgos/plugins/PATMuonSlimmer.cc

```diff
@@ -56,6 +61,8 @@ pat::PATMuonSlimmer::PATMuonSlimmer(const edm::ParameterSet & iConfig) :
         for (const edm::InputTag &tag : pf2pc) pf2pc_.push_back(consumes<edm::Association<pat::PackedCandidateCollection>>(tag));
     }
 
+    pc_ = consumes<pat::PackedCandidateCollection>(iConfig.getParameter<std::vector<edm::InputTag>>("packedPFCandidates")[0]);
```

Comment:
I added an input tag specifically for the mini-iso computation to avoid confusion.